composer/utils/checkpoint.py

```diff
@@ -566,6 +567,23 @@ def dist_cp_load(
                 planner=load_planner,
             )
         except CheckpointException as e:
+            if re.search(r'Size mismatch.+torch.Size([816]).+rng.+cuda', str(e)) is not None:
+                # In torch 2.6, Pytorch (throughout) is strictly enforcing the sizes of the
```

Comment:
lol the "throughout" meant that you should change "we" to "PyTorch" in this whole comment. Not meant "throughout" as part of the comment